src/playwright/transformPlaywrightJson.test.ts

```diff
@@ -198,6 +207,165 @@ describe('Playwright Json', () => {
       `);
     });
 
+    it('should respect include, exclude and skip tags', () => {
+      process.env.STORYBOOK_INCLUDE_TAGS = 'play,design';
+      process.env.STORYBOOK_SKIP_TAGS = 'skip';
+      process.env.STORYBOOK_EXCLUDE_TAGS = 'exclude';
+      const input = {
+        v: 4,
+        entries: {
+          A: {
+            id: 'example-a',
+            title: 'Example/Header',
+            name: 'Logged In',
+            importPath: './stories/basic/Header.stories.js',
+            tags: ['play', 'exclude'],
+          },
+          B: {
+            id: 'example-b',
+            title: 'Example/Header',
+            name: 'Logged Out',
+            importPath: './stories/basic/Header.stories.js',
+            tags: ['play', 'skip'],
+          },
+          C: {
+            id: 'example-c',
+            title: 'Example/Page',
+            name: 'Logged In',
+            importPath: './stories/basic/Page.stories.js',
+            tags: ['design'],
+          },
+          D: {
+            id: 'example-d',
+            title: 'Example/Page',
+            name: 'Logged In',
+            importPath: './stories/basic/Page.stories.js',
+          },
+        },
+      };
+      // Should result in:
+      // - A being excluded
+      // - B being included, but skipped
+      // - C being included
+      // - D being excluded
+      expect(transformPlaywrightJson(input)).toMatchInlineSnapshot(`
```

Comment:
I'm not a huge fan of this huge snapshot tbh.
splitting this into 2 files: input & output and then asserting with given input, the output matches the file would make it easier to manually check if the output doesn't contain some small mistake.

There's 0 syntax safety here.